ChangeLog

```diff
@@ -12,6 +12,9 @@ What's New in astroid 2.11.3?
 =============================
 Release date: TBA
 
+* Fixed an error in the Qt brain when building ``instance_attrs``.
+
+  Closes PyCQA/pylint#6221
```

Comment:
I can move this to 2.12, but I was wondering if we will be releasing it before pylint 2.14 to prevent user crashes.

Reply:
Sorry oversight on my part, it should be the next patch release indeed.